{"name": "使用无效密码登录", "status": "passed", "description": "测试使用无效密码登录", "steps": [{"name": "发送登录请求", "status": "passed", "start": 1742985835909, "stop": 1742985835911}, {"name": "验证登录失败", "status": "passed", "start": 1742985835911, "stop": 1742985835911}], "attachments": [{"name": "stderr", "source": "61ba8715-30d0-46e0-b778-e97d58db1ea0-attachment.txt", "type": "text/plain"}], "start": 1742985835909, "stop": 1742985835911, "uuid": "ce5a11a3-ddbf-4fdb-be22-04e1ea12ce46", "historyId": "67369e25bc2f201587578f9a247db44d", "testCaseId": "67369e25bc2f201587578f9a247db44d", "fullName": "test_cases.api.test_demo_api.TestUserAPI#test_login_with_invalid_password", "labels": [{"name": "epic", "value": "API测试"}, {"name": "story", "value": "登录接口"}, {"name": "feature", "value": "用户管理"}, {"name": "parentSuite", "value": "test_cases.api"}, {"name": "suite", "value": "test_demo_api"}, {"name": "subSuite", "value": "TestUserAPI"}, {"name": "host", "value": "MacBook-Pro.local"}, {"name": "thread", "value": "39379-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.api.test_demo_api"}]}